filebeat/module/system/auth/test/test.log-expected.json

```diff
@@ -1,12 +1,11 @@
 [
     {
         "ecs.version": "1.0.0",
+        "event.action": "ssh_login",
```

Comment:
Seems like these didn't get updated in a while since the ordering was off.

Reply:
I'm wondering, why were tests not failing then?

Reply:
The tests were not failing because the content is all there, just the order was different. When comparing we compare the entries and not the order of the fields.